README.md

```diff
@@ -74,6 +77,24 @@ spring.mail.password=$PASSWORD
 All other `spring.mail.*` configurations can be found in the [Spring Documentation E-Mail](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#io.email)
 be viewed.
 
+#### Security Provider
+
+As security provider OIDC-based security providers are possible to use (e.g. [Keycloak](https://www.keycloak.org/) or Microsoft Azure AD).
+To configure the security provider, the following configurations must be made.
```

Comment:
as zeiterfassung is using user permissions from oidc claims, maybe it's a good idea to describe the `groups` claim and possible permissions here